public/src/components/Record.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  faTint, faPoop, faBed, faSun, faSpinner,
+} from '@fortawesome/free-solid-svg-icons';
+
+export const StyledRecord = styled.div`
+  ${({ theme, accident }) => css`
+    border-radius: ${theme.borderRadius};
+    background-color: ${accident ? theme.flavors.accident : theme.flavors.record};
+    display: flex;
+    flex-direction: column;
+    padding: .5em 1em;
+    color: ${theme.flavors.background};
+  `}
+`;
+export const RecordIconRack = styled.div`
+  display: flex;
+  padding: .25em 0;
+  svg + svg {
+    margin-left: 1em;
+  }
+`;
+export const RecordFAIcon = styled(FontAwesomeIcon)`
+  ${({ active }) => css`
+    path {
+      fill: white;
+    }
+    ${active ? css`` : css`
+      opacity: .4;
+    `}
+  `}
+`;
+
+export const Record = ({ data }) => {
+  if (typeof data === 'undefined') {
```

Comment:
Shouldn't ever be undefined if data is required

You either have bad prop types or bad consumers if this is happening